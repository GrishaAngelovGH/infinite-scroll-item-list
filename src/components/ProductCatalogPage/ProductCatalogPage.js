import Layout from 'components/Layout'
import ProductCatalog from 'components/ProductCatalog'

const ProductCatalogPage = () => {
  return (
    <Layout
      content={<ProductCatalog />}
    />
  )
}

export default ProductCatalogPage