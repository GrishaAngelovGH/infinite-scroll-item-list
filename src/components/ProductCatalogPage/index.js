import ProductCatalogPage from './ProductCatalogPage'

export default ProductCatalogPage