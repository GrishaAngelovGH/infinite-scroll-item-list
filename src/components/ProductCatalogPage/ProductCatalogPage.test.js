import { render } from '@testing-library/react'

import ProductCatalogPage from './ProductCatalogPage'

test('should render ProductCatalogPage component', () => {
  const view = render(<ProductCatalogPage />)

  expect(view).toMatchSnapshot()
})