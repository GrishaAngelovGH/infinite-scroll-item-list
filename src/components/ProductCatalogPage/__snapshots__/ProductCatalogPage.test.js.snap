// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ProductCatalogPage component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="ant-layout ant-layout-has-sider css-dev-only-do-not-override-yp8pcc"
        style="height: 100vh;"
      >
        <aside
          class="ant-layout-sider ant-layout-sider-dark"
          style="text-align: center; line-height: 120px; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray; flex: 0 0 200px; max-width: 200px; min-width: 200px; width: 200px;"
        >
          <div
            class="ant-layout-sider-children"
          >
            Sider
          </div>
        </aside>
        <section
          class="ant-layout css-dev-only-do-not-override-yp8pcc"
        >
          <header
            class="ant-layout-header css-dev-only-do-not-override-yp8pcc"
            style="text-align: center; height: 64px; padding-inline: 50px; line-height: 64px; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray;"
          >
            Header
          </header>
          <main
            class="ant-layout-content css-dev-only-do-not-override-yp8pcc"
            style="text-align: center; min-height: 120px; line-height: 120px; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray;"
          >
            <div>
              ProductCatalog
            </div>
          </main>
          <footer
            class="ant-layout-footer css-dev-only-do-not-override-yp8pcc"
            style="text-align: center; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray;"
          >
            Footer
          </footer>
        </section>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="ant-layout ant-layout-has-sider css-dev-only-do-not-override-yp8pcc"
      style="height: 100vh;"
    >
      <aside
        class="ant-layout-sider ant-layout-sider-dark"
        style="text-align: center; line-height: 120px; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray; flex: 0 0 200px; max-width: 200px; min-width: 200px; width: 200px;"
      >
        <div
          class="ant-layout-sider-children"
        >
          Sider
        </div>
      </aside>
      <section
        class="ant-layout css-dev-only-do-not-override-yp8pcc"
      >
        <header
          class="ant-layout-header css-dev-only-do-not-override-yp8pcc"
          style="text-align: center; height: 64px; padding-inline: 50px; line-height: 64px; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray;"
        >
          Header
        </header>
        <main
          class="ant-layout-content css-dev-only-do-not-override-yp8pcc"
          style="text-align: center; min-height: 120px; line-height: 120px; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray;"
        >
          <div>
            ProductCatalog
          </div>
        </main>
        <footer
          class="ant-layout-footer css-dev-only-do-not-override-yp8pcc"
          style="text-align: center; color: rgb(0, 0, 0); background-color: rgb(255, 255, 255); border: 3px solid gray;"
        >
          Footer
        </footer>
      </section>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
